benchmark/cajita/CMakeLists.txt

```diff
@@ -28,6 +31,9 @@ if(Kokkos_ENABLE_SERIAL OR Kokkos_ENABLE_CUDA OR Kokkos_ENABLE_OPENMP OR Kokkos_
     target_link_libraries(SparsePartitionerPerformance gtest)
     add_test(NAME Cajita_SparsePartitionerPerformance COMMAND ${NONMPI_PRECOMMAND} SparsePartitionerPerformance sparsepartitioner_output.txt)
 
+    target_link_libraries(HaloPerformance gtest)
```

Comment:
Why would you link `gtest`?

Reply:
Yeah, no idea why we're doing that for all these cases - Kwitae we can remove this line

Reply:
removed